cdc/owner.go

```diff
@@ -105,20 +106,21 @@ func (c *changeFeed) String() string {
 	return s
 }
 
-func (c *changeFeed) updateProcessorInfos(processInfos model.ProcessorsInfos) {
-	for cid, pinfo := range processInfos {
+func (c *changeFeed) updateProcessorInfos(processInfos model.ProcessorsInfos, position map[string]*model.TaskPosition) {
+	for cid, pinfo := range position {
```

Comment:
recommend to refine variable names in this function
s/position/positions
s/pinfo/position
s/oldPinfo/oldPosition